src/api/needs-assessment/content-types/need/schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "kind": "collectionType",
+  "collectionName": "needs",
+  "info": {
+    "singularName": "need",
+    "pluralName": "needs",
+    "displayName": "NeedsAssessment.Need"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "needs_assessment_survey": {
```

Comment:
**suggestion(non-blocking):** It would be cleaner to just use the `type` name for fields, instead of `api_type`.  I'm thinking about how we access these properties in the frontend code later on: `need.survey` vs. `need.needs_assessment_survey`.  This applies to the other fields as well (`geo_region`, etc).